daprdocs/content/en/js-sdk-docs/js-server/_index.md

```diff
@@ -228,6 +228,49 @@ async function start() {
 
 > For a full list of state operations visit [How-To: Publish & subscribe]({{< ref howto-publish-subscribe.md >}}).
 
+#### Subscribe with SUCCESS/RETRY/DROP status
+
+Dapr supports [status codes for retry logic](https://docs.dapr.io/reference/api/pubsub_api/#expected-http-response) to specify what should happen after a message gets processed.
+
+> ⚠️ The JS SDK allows multiple callbacks on the same topic, we handle priority of status on `RETRY` > `DROP` > `SUCCESS` and default to `SUCCESS`
+
+In the JS SDK we support these messages through the `DaprPubSubStatusEnum` enum:
+
+```typescript
+import { DaprServer, DaprPubSubStatusEnum } from "@dapr/dapr";
+
+const daprHost = "127.0.0.1"; // Dapr Sidecar Host
+const daprPort = "3500"; // Dapr Sidecar Port of this Example Server
+const serverHost = "127.0.0.1"; // App Host of this Example Server
+const serverPort = "50051"; // App Port of this Example Server "
+
+async function start() {
+  const server = new DaprServer(serverHost, serverPort, daprHost, daprPort);
+
+  const pubSubName = "my-pubsub-name";
+  const topic = "topic-a";
+
+  // Process a message successfully
+  await server.pubsub.subscribe(pubSubName, topic, async (data: any, headers: object) => {
+    return DaprPubSubStatusEnum.SUCCESS;
+  });
+
+  // Retry a message
+  // Note: this example will keep on retrying to deliver the message
```

Comment:
Should we add a note that this is component specific?

Reply:
How do you mean component specific?

Reply:
So components can have their own retry configurations, e.g., https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-redis-pubsub/, `redisMaxRetries` is 0 by default.

Reply:
Ok added a note